src/main/java/com/applicaster/onboarding/screen/OnboardingScreenContract.java

```diff
@@ -35,13 +36,13 @@ public void setPluginConfigurationParams(Map params) {
 
     @Override
     public void executeOnApplicationReady(final Context context, final HookListener listener) {
-        String[] selections = PreferenceUtil.getInstance().getStringArrayPref("user_ob_selections", null);
+        String[] selections = PreferenceUtil.getInstance().getStringArrayPref("userRecommendationTags", null);
```

Comment:
`userRecommendationTags` I think it will be good to have this string in a constant just in case it changes in the future.